charts/backstage/README.md

```diff
@@ -0,0 +1,261 @@
+# Backstage Helm Chart
+
+![Version: 0.5.4](https://img.shields.io/badge/Version-0.5.4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+
+A Helm chart for deploying a Backstage application
+
+---
+
+[Backstage](https://backstage.io) is an open platform for building developer portals. Powered by a centralized software catalog, Backstage restores order to your microservices and infrastructure and enables your product teams to ship high-quality code quickly — without compromising autonomy.
+
+Backstage unifies all your infrastructure tooling, services, and documentation to create a streamlined development environment from end to end.
+
+## TL;DR
+
+```console
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add backstage https://backstage.github.io/charts
+
+helm install my-release backstage/backstage
+```
+
+## Introduction
+
+This chart bootstraps a [Backstage](https://backstage.io/docs/deployment/docker) deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.2.0+
+- PV provisioner support in the underlying infrastructure
+- [Backstage container image](https://backstage.io/docs/deployment/docker)
+
+## Usage
+
+Chart is available in the following formats:
+
+- [Chart Repository](https://helm.sh/docs/topics/chart_repository/)
+- [OCI Artifacts](https://helm.sh/docs/topics/registries/)
+
+### Installing from the Chart Repository
+
+The following command can be used to add the chart repository:
+
+```console
+helm repo add backstage https://backstage.github.io/charts
+helm install my-release backstage/backstage
```

Comment:
Similar thing here, do we want to put the install command in this code snippet if the description only says _"The following command can be used to add the chart repository:"_? We have the install command in the next snippet down

Reply:
I agree removing the `helm install ...` line.